File: C/mario.c
#include <stdio.h>
#include <cs50.h>

int get_height(string prompt);
int main(void)
{
    int i = get_height("Height: ");
    for (int e = 1; e <= i; e++)
        {
               for (int q = (i - e); q > 0; q--)
               {
               printf(" ");
               }
                    for (int w = 0; w < e; w++)
                    {
                        printf("#");
                    }
        printf("\n");
}
        }
        

int get_height(string prompt)
{
    int n;
    do
    {
        n = get_int("%s", prompt);
    }
    while (n > 8 || n < 1);
    return n;
}

File: Python/mario.py
# реалізація get_int 
while True:
	try:
		n = int(input('Height: '))
	except ValueError:
		pass
	else:	
		if n > 0 and n < 9:
			break

# сам print
for i in range(n):
	print(" " * (n-1), end="")
	print("#" * (i+1), end="") # друкує '#' без вирівнювання
	print("  ", end="")
	print("#" * (i+1)) # друкує '#' без вирівнювання

	n-=1
